ResponseDetective/Sources/URLEncodedBodyDeserializer.swift

```diff
@@ -0,0 +1,42 @@
+//
+// URLEncodedBodyDeserializer.swift
+//
+// Copyright © 2016-2017 Netguru Sp. z o.o. All rights reserved.
+// Licensed under the MIT License.
+//
+
+import Foundation
+
+/// Deserializes URL-encoded bodies.
+@objc(RDTURLEncodedBodyDeserializer) public final class URLEncodedBodyDeserializer: NSObject, BodyDeserializer {
+
+	// MARK: BodyDeserializer
+
+	/// Deserializes JSON data into a pretty-printed string.
+	public func deserialize(body: Data) -> String? {
+
+		guard let string = String(data: body, encoding: .utf8) else {
+			return nil
+		}
+
+		if #available(macOS 10.10, tvOS 10.0, *) {
+
+			let stringFromQueryItems: ([URLQueryItem]) -> String = {
+				$0.map { "\($0.name): \($0.value ?? "nil")" }.joined(separator: "\n")
+			}
+
+			if let items = URLComponents(string: string)?.queryItems {
```

Comment:
Hell, I even fell into the trap and, since I'm not familiar with tvOS, I marked this as available since tvOS 10.0, while it's _really_ available since tvOS 9.0 (the first release of tvOS). 🙈